salt/modules/rpm.py

```diff
@@ -477,17 +498,23 @@ def info(*packages):
             if len(line) != 2:
                 continue
             key, value = line
-            key = key.replace(' ', '_').lower()
             if key == 'description':
                 descr_marker = True
                 continue
             if key == 'name':
                 pkg_name = value
             if key in ['build_date', 'install_date']:
-                value = _pkg_time_to_iso(value)
-            if key != 'description' and value:
+                try:
+                    value = int(value)
+                    pkg_data['{0}_time_t'.format(key)] = value
+                    value = datetime.datetime.fromtimestamp(value).isoformat() + "Z"
```

Comment:
So if I ask for one attribute (e.g. `build_date`), I get two attributes (`build_date` and `build_date_time_t`) back ?!
Not very consistent.